### PR TITLE
Remove tibble

### PR DESCRIPTION
**Pull request by jdblischak**
This PR removes {tibble} from the package dependencies. Note however that all the functions can still accept {tibble} as input, so all existing code is still valid

### CODE REVIEW

File: R/MBdelayed.R [hunk 1]
```diff
@@ -53,7 +53,7 @@
 #'
 #' # MaxCombo with logrank, FH(0,1), FH(1,1)
 #' ten %>%
-#'   fh_weight(rho_gamma = tibble(rho = c(0, 0, 1), gamma = c(0, 1, 1)), return_corr = TRUE) %>%
```
Comment (LittleBeannie) [inline R/MBdelayed.R:56]: If users input `tibble` rather than `data.frame`, can they still run the codes smoothly?

**Review by LittleBeannie — COMMENTED** [review]
Hi John, I have 1 question below. Could you please help with it?

File: R/MBdelayed.R (hunk 1 above)
Comment (jdblischak) [reply R/MBdelayed.R:56]: Correct. The code still runs smoothly if the input is a data frame. Tibbles are enhanced data frames, but ultimately are still data frames, which you can see from their classes, eg

```R
class(tibble::tibble(mtcars))
## [1] "tbl_df"     "tbl"        "data.frame"
```

Comment (LittleBeannie) [reply R/MBdelayed.R:56]: Oh, maybe I am not very clear on my question. Can the code be run if the input is a tibble?

Comment (jdblischak) [reply R/MBdelayed.R:56]: Sorry for the unclear answer. All the functions accept either tibbles or data frames. The output will always be identical.

To be explicit, the code below demonstrates that a tibble or data frame input does not affect the returned result:

```R
identical(
  {set.seed(1); fh_weight(rho_gamma = data.frame(rho = c(0, 0, 1, 1), gamma = c(0, 1, 0, 1)))},
  {set.seed(1); fh_weight(rho_gamma = tibble::tibble(rho = c(0, 0, 1, 1), gamma = c(0, 1, 0, 1)))}
)
## TRUE
```

Comment (LittleBeannie) [reply R/MBdelayed.R:56]: Thanks for explaining this to me!

**Review by nanxstats — APPROVED** [review]
This is a small but an important step - appreciate the change!

**Pull request merged**